### PR TITLE
Add the `eventloopIdleDelaySec` capability (wdaEventloopIdleDelay for users)

### PR DESCRIPTION
**Pull request by dmissmann**
Add support for handling the `eventloopIdleDelaySec` capability in WDA (https://github.com/appium/WebDriverAgent/pull/140).

### CODE REVIEW

File: lib/driver.js [hunk 1]
```diff
@@ -844,6 +844,7 @@ class XCUITestDriver extends BaseDriver {
     let maxTypingFrequency = util.hasValue(this.opts.maxTypingFrequency) ? this.opts.maxTypingFrequency : 60;
     let shouldUseSingletonTestManager = util.hasValue(this.opts.shouldUseSingletonTestManager) ? this.opts.shouldUseSingletonTestManager : true;
     let shouldUseTestManagerForVisibilityDetection = false;
+    let eventloopIdleDelaySec = util.hasValue(this.opts.eventloopIdleDelaySec) ? this.opts.eventloopIdleDelaySec : 0;
```
Comment (mykola-mokhnach) [inline lib/driver.js]: this.opts.eventloopIdleDelaySec || 0

File: README.md [hunk 2]
```diff
@@ -162,6 +162,7 @@ Differences noted here
...
 |`skipLogCapture`|Skips to start capturing logs such as crash, system, safari console and safari network. It might improve performance such as network. Log related commands will not work. Defaults to `false`. |`true` or `false`|
+|`eventloopIdleDelaySec`|Delays the invocation of `-[XCUIApplicationProcess setEventLoopHasIdled:]` by the number of seconds specified with this capability. This can help quiescence apps that fail to do so for no obvious reason (and creating a session fails for that reason). This increases the time for session creation because `-[XCUIApplicationProcess setEventLoopHasIdled:]` is called multiple times. If you enable this capability start with at least `3` seconds and try increasing it, if creating the session still fails. Defaults to `0`. |e.g. `5`|
```
Comment (KazuCocoa) [inline README.md]: 👍

**Comment by imurchie** [comment]
I think the name of this is confusing. I would add a WDA in there somewhere, to distinguish from, say, the Node event loop.

File: lib/desired-caps.js [hunk 3]
```diff
@@ -168,6 +168,9 @@ let desiredCapConstraints = _.defaults({
   skipLogCapture: {
     isBoolean: true
   },
+  eventloopIdleDelaySec: {
```
Comment (KazuCocoa) [inline lib/desired-caps.js]: `wdaEventloopIdleDelay` here as README?

File: lib/driver.js [hunk 4]
```diff
@@ -844,6 +844,7 @@ class XCUITestDriver extends BaseDriver {
     let maxTypingFrequency = util.hasValue(this.opts.maxTypingFrequency) ? this.opts.maxTypingFrequency : 60;
     let shouldUseSingletonTestManager = util.hasValue(this.opts.shouldUseSingletonTestManager) ? this.opts.shouldUseSingletonTestManager : true;
     let shouldUseTestManagerForVisibilityDetection = false;
+    let wdaEventloopIdleDelay = this.opts.eventloopIdleDelaySec || 0;
```
Comment (KazuCocoa) [inline lib/driver.js]: `let eventloopIdleDelaySec = this.opts. wdaEventloopIdleDelay || 0` and `eventloopIdleDelaySec` as a desired capability in create session below?

File: lib/desired-caps.js (hunk 3 above)
Comment (dmissmann) [reply lib/desired-caps.js]: Yes of course. Mixed the two names up...

**Comment by dmissmann** [comment]
@KazuCocoa the commit for the WDA submodule was already bumped to the most recent commit. And the required changes were already merged here: https://github.com/appium/WebDriverAgent/commit/5896cce5a3ae386cf1874f3554ca462861a904df so there's no need to do anything else here

**Pull request merged**